Documentation/async-disaster-recovery.md

```diff
@@ -0,0 +1,454 @@
+
+```bash
+    kubectl create -f pv-backup.yaml --context=cluster-2
+```
+
+> ```bash
+> persistentvolume/pvc-65dc0aac-5e15-4474-90f4-7a3532c621ec created
+> ```
+
+* Apply the Persistent Volume claim from the restored backup
+
+```bash
+    kubectl create -f pvc-backup.yaml --context=cluster-2
+```
+
+> ```bash
+> persistentvolumeclaim/rbd-pvc created
+> ```
+
+```bash
+  kubectl get pvc --context=cluster-2
+```
+
+> ```bash
+> NAME      STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS      AGE
+> rbd-pvc   Bound    pvc-65dc0aac-5e15-4474-90f4-7a3532c621ec   1Gi        RWO            csi-rbd-sc   44s
+> ```
+
+## Planned Migration
+
+> Use cases: Datacenter maintenance, Technology refresh, Disaster avoidance, etc.
+
+### Failover
+
+The failover operation is the process of switching production to a
+ backup facility (normally your recovery site). In the case of Failover,
+ access to the image on the primary site should be stopped.
+The image should now be made *primary* on the secondary cluster so that
+ the access can be resumed there.
+
+:memo: As mentioned in the pre-requisites, periodic or one time backup of
+ the application should be available for restore on the secondary site (cluster-b).
+
+Follow the below steps for planned migration of workload from primary
+ cluster to secondary cluster:
+
+* Scale down all the application pods which are using the
+ mirrored PVC on the Primary Cluster
+* Take a back up of PVC and PV object from the primary cluster.
+ This can be done using some backup tools like
+ [velero](https://velero.io/docs/main/).
+* Update `replicationState` to `secondary` in VolumeReplication CR at Primary Site.
+ When the operator sees this change, it will pass the information down to the
+  driver via GRPC request to mark the dataSource as `secondary`.
+* If you are manually recreating the PVC and PV on the secondary cluster,
+ remove the `claimRef` section in the PV objects.
+* Recreate the storageclass, PVC, and PV objects on the secondary site.
+* As you are creating the static binding between PVC and PV, a new PV won’t
+ be created here, the PVC will get bind to the existing PV.
+* Create the VolumeReplicationClass on the secondary site.
+* Create the VolumeReplications for all the PVC’s for which mirroring
+ is enabled
+  * `replicationState` should be `primary` for all the PVC’s on
+   the secondary site.
+* Check whether the image is marked `primary` on the secondary site
+ by verifying it in VolumeReplication CR status.
+* Once the Image is marked as `primary`, the PVC is now ready
+ to be used. Now, we can scale up the applications to use the PVC.
+
+>:memo: **WARNING**: In Async Disaster recovery use case, we don't get
+> the complete data.
+> We will only get the crash-consistent data based on the snapshot interval time.
+
+### Failback
+
+To perform a failback operation to primary cluster in case of planned migration
+, just repeat the Failback steps in vice-versa.
```

Comment:
We should give the steps (with `kubectl` commands)  rather than expecting users to follow the vice-versa steps.  This is just my opinion.